web/conf.py

```diff
@@ -178,7 +178,9 @@
                 # (TELECON_LINK, "Teleconference room (Jitsi Meet)"),
                 (TELECON_CALENDAR, "Calendar"),
                 ("/meetings", "Meetings"),
+                (FACEBOOK, "Facebook"),
                 (TWITTER, "Twitter"),
+                (YOUTUBE_CHANNEL, "YouTube"),
```

Comment:
By the way, a cool thing I recently realized - you can put images, logos etc in here.

Reply:
I was wondering about that! I would like to put the social media icons there. Just need to make sure we have the rights to use them.